rfcs/text/0025-container-metric-fields.md

```diff
@@ -329,6 +329,483 @@ Stage 3: Add more real world example source documents so we have at least 2 tota
+      },
+      {
+        "major": 253,
+        "minor": 0,
+        "op": "Total",
+        "value": 2404
+      }
+    ],
+    "io_time_recursive": [],
+    "io_wait_time_recursive": [],
+    "sectors_recursive": []
+  },
+  "cpu_stats": {
+    "cpu_usage": {
+      "percpu_usage": [
+        8989285945449,
+        8802991575246,
+        8723071634953,
+        8812077512624
+      ],
+      "total_usage": 35327426668272,
+      "usage_in_kernelmode": 2720580000000,
+      "usage_in_usermode": 32477900000000
+    },
+    "online_cpus": 4,
+    "system_cpu_usage": 7640294590000000,
+    "throttling_data": {
+      "periods": 0,
+      "throttled_periods": 0,
+      "throttled_time": 0
+    }
+  },
+  "id": "86a4d10d07bf0b1ded54c7156b4362e8b0dba9d450201a9695b4714c1492829c",
+  "memory_stats": {
+    "limit": 15620788224,
+    "max_usage": 420536320,
+    "stats": {
+      "active_anon": 59502592,
+      "active_file": 63746048,
+      "cache": 194330624,
+      "dirty": 0,
+      "hierarchical_memory_limit": 9223372036854771712,
+      "hierarchical_memsw_limit": 9223372036854771712,
+      "inactive_anon": 60268544,
+      "inactive_file": 130256896,
+      "mapped_file": 123408384,
+      "pgfault": 37852947,
+      "pgmajfault": 0,
+      "pgpgin": 25772802,
+      "pgpgout": 25697155,
+      "rss": 119738368,
+      "rss_huge": 4194304,
+      "total_active_anon": 59502592,
+      "total_active_file": 63746048,
+      "total_cache": 194330624,
+      "total_dirty": 0,
+      "total_inactive_anon": 60268544,
+      "total_inactive_file": 130256896,
+      "total_mapped_file": 123408384,
+      "total_pgfault": 37852947,
+      "total_pgmajfault": 0,
+      "total_pgpgin": 25772802,
+      "total_pgpgout": 25697155,
+      "total_rss": 119738368,
+      "total_rss_huge": 4194304,
+      "total_unevictable": 0,
+      "total_writeback": 0,
+      "unevictable": 0,
+      "writeback": 0
+    },
+    "usage": 326488064
+  },
+  "name": "/elastic-package-stack_elastic-agent_1",
+  "networks": {
+    "eth0": {
+      "rx_bytes": 23962833,
+      "rx_dropped": 0,
+      "rx_errors": 0,
+      "rx_packets": 128811,
+      "tx_bytes": 593646535,
+      "tx_dropped": 0,
+      "tx_errors": 0,
+      "tx_packets": 186477
+    }
+  },
+  "num_procs": 0,
+  "pids_stats": {
+    "current": 68
+  },
+  "precpu_stats": {
+    "cpu_usage": {
+      "percpu_usage": [
+        8989283978798,
+        8802978152064,
+        8723042900675,
+        8812068229234
+      ],
+      "total_usage": 35327373260771,
+      "usage_in_kernelmode": 2720550000000,
+      "usage_in_usermode": 32477870000000
+    },
+    "online_cpus": 4,
+    "system_cpu_usage": 7640290600000000,
+    "throttling_data": {
+      "periods": 0,
+      "throttled_periods": 0,
+      "throttled_time": 0
+    }
+  },
+  "preread": "2021-02-04T22:02:38.129030814Z",
+  "read": "2021-02-04T22:02:39.138448063Z",
+  "storage_stats": {}
+}
+```
+
+#### Mapped event by docker.cpu dataset
+
```

Comment:
Should we add a paragraph before each sample event mentioning the relevant fields to this RFC?

For example here it could be something like this:
 ```suggestion

Event contains `container` fields identifying the container and its runtime. This sample event also contains a summary of the CPU metrics in `container.cpu`.
```